map_matching/js/map_matching.js

```diff
@@ -68,9 +68,125 @@
         return L.geoJson(geo, options);
     };
 
+    var encode = function (coordinates) {
```

Comment:
the following large green block of additions is the main point of this pull request to do the marshalling of json objects between old format and new